src/index.html

```diff
@@ -92,53 +81,96 @@
         <div style="display: block; margin-bottom: 10px">
           <span style="color: #444; font-size: 14px; margin-bottom: 4px; display: block">
             <strong>Project Founded and Directed by:</strong>
-            Daniel Glejzner (Angular Bros) 
-            <a href="https://x.com/danielglejzner" target="_blank" rel="nofollow" style="color: #444; text-decoration: none">
+            Daniel Glejzner (Angular Bros)
+            <a
+              href="https://x.com/danielglejzner"
+              aria-label="Daniel Glejzner account on X"
+              target="_blank"
+              rel="nofollow"
+              style="color: #444; text-decoration: none"
```

Comment:
move these styles to css file

Reply:
Same as abose, this would fit more in a refactor in another PR.

Also, this CSS inlining should be taken in consideration, if it's not inlined it could be delayed by the loading of the style file and idk if it could impact SEO concerns, if the footer is directly in index.html, I think there's a reason (I myself don't know why haha)

Reply:
I don't think there is any sophisticated thought backing this up it's just me being lazy and wanted to deliver this asap today 😂

Reply:
> Same as abose, this would fit more in a refactor in another PR.
> 
> Also, this CSS inlining should be taken in consideration, if it's not inlined it could be delayed by the loading of the style file and idk if it could impact SEO concerns, if the footer is directly in index.html, I think there's a reason (I myself don't know why haha)

Of course, inline styles have the best performance, but I think that doesn't affect too much 😄